unittests/allplatformstests.py

```diff
@@ -4440,7 +4442,15 @@ def output_name(name, type_):
             'install_subdirs': {
                 f'{testdir}/custom_files': {
                     'destination': '{datadir}/custom_files',
-                    'tag': 'custom'
+                    'tag': 'custom',
+                    'exclude_dirs': [],
+                    'exclude_files': [],
+                },
+                f'{testdir}/excludes': {
+                    'destination': '{datadir}/excludes',
+                    'tag': 'custom',
+                    'exclude_dirs': ['excluded'],
```

Comment:
The argument name is `exclude_directories` while most code use `exclude_dirs` as a variable name. Should the filed in the JSON be named `exclude_directories` or `exclude_dirs`? I went for the short options, but maybe consistency with the the argument name is desirable?

Reply:
@eli-schwartz Any thought on the naming of these fields?

Reply:
I think the code is an internal implementation detail for conciseness, and there's no particular reason to cleave to it.

Reply:
Sure, but do you like concise filed names in the introspection data or do you prefer introspection data to be named as close as possible to the Meson's DSL option names? I don't have a preference either way.

Reply:
Well... nor do I. :D